pkg/networkservice/chains/nsmgr/single_test.go

```diff
@@ -254,3 +255,53 @@ func Test_Local_NoURLUsecase(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, int32(1), atomic.LoadInt32(&counter.Closes))
 }
+
+func Test_ShouldParseNetworkServiceLabelsTemplate(t *testing.T) {
+	t.Cleanup(func() { goleak.VerifyNone(t) })
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+
+	const (
+		testEnvName             = "NODE_NAME"
+		testEnvValue            = "testValue"
+		destinationTestKey      = "nodeName"
+		destinationTestTemplate = "{{index .Src \"NodeNameKey\"}}"
```

Comment:
It would probably be better to use raw string literals in such cases:
```suggestion
		destinationTestTemplate = `{{index .Src "NodeNameKey"}}`
```
@Bolodya1997 @denis-tingaikin do you approve?

Reply:
Yes, I guess so.

Reply:
I think it is a cosmetic thing and if the linter is not minded then we can use `"{{index .Src \"NodeNameKey\"}}"`